codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/Encoder.java

```diff
@@ -351,10 +360,6 @@ private void add(byte[] name, byte[] value) {
             remove();
         }
 
-        // Copy name and value that modifications of original do not affect the dynamic table.
```

Comment:
@Scottmitch let me do a follow-up to also protect our headers. I think I have an idea :)